docs/cql/ddl.rst

```diff
@@ -769,12 +769,7 @@ available:
                                            for a read. Allowed values are powers of two between 1 and 128.
 ========================= =============== =============================================================================
 
-.. ``crc_check_chance``      1.0             When compression is enabled, each compressed block includes a checksum of
-..                                           that block for the purpose of detecting disk bitrot and avoiding the
-..                                           propagation of corruption to other replicas. This option defines the
-..                                           probability with which those checksums are checked during read. By default
-..                                           they are always checked. Set to 0 to disable checksum checking and to 0.5 for
-..                                           instance to check them every other read   |
+.. crc_check_chance is ignored and not supported in ScyllaDB
```

Comment:
"Not supported" may suggest we raise an error if user provides a value for this option, which is not true, we simply ignore it.

Note that Cassandra promoted this option to top-level, so it applies to both compressed and uncompressed sstables, and it is deprecated in the compression map.

Reply:
i see. will update the comment to reflect these information. and repurpose this change to claim that it is *ignored* like #21795